src/routes/(marketing)/blog/(posts)/+layout.svelte

```diff
@@ -16,14 +17,30 @@
   if (!currentPost) {
     throw error(404, "Blog post not found")
   }
+
+  const pageTitle = currentPost?.title ? currentPost.title : "Not Found"
+  const pageDescription = currentPost?.description ? currentPost.description : "Blog post"
+  const pageUrl = $page.url.origin + $page.url.pathname
```

Comment:
An alternative would be `window.location.href` but that's not available in SSR.
Another alternative is to use `$page.url` but that might looks like this `https://example.com/test/test2/test3#hash?test=test`
Doing the above will give a nice looking URL as follow: `https://example.com/test/test2/test3`